PennMobile/Setup + Navigation/AppDelegate+NotificationExtension.swift

```diff
@@ -28,7 +28,13 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
         }
 
         let token = tokenParts.joined()
-        UserDBManager.shared.savePushNotificationDeviceToken(deviceToken: token)
+
+        UserDBManager.shared.getNotificationId() { result in
```

Comment:
I believe you wouldn't need the trailing `()`, in swift, if your last variable is a closure, you wouldn't need the closing brackets.